File: sdkwork-app-sdk-go/types/splash_advert_vo.go
package types

// 开屏广告VO
type SplashAdvertVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	AdvertId string `json:"advertId"`
	AdvertName string `json:"advertName"`
	AdvertType string `json:"advertType"`
	PositionId string `json:"positionId"`
	CreativeUrl string `json:"creativeUrl"`
	LandingUrl string `json:"landingUrl"`
	Priority int `json:"priority"`
	Enabled bool `json:"enabled"`
	StartTime string `json:"startTime"`
	EndTime string `json:"endTime"`
	ClickCount int `json:"clickCount"`
	ImpressionCount int `json:"impressionCount"`
	DisplayDuration int `json:"displayDuration"`
	SkipEnabled bool `json:"skipEnabled"`
	SkipDelay int `json:"skipDelay"`
	BackgroundColor string `json:"backgroundColor"`
	LogoUrl string `json:"logoUrl"`
}

File: sdkwork-app-sdk-csharp/Models/SplashAdvertVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class SplashAdvertVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? AdvertId { get; set; }
        public string? AdvertName { get; set; }
        public string? AdvertType { get; set; }
        public string? PositionId { get; set; }
        public string? CreativeUrl { get; set; }
        public string? LandingUrl { get; set; }
        public int? Priority { get; set; }
        public bool? Enabled { get; set; }
        public string? StartTime { get; set; }
        public string? EndTime { get; set; }
        public int? ClickCount { get; set; }
        public int? ImpressionCount { get; set; }
        public int? DisplayDuration { get; set; }
        public bool? SkipEnabled { get; set; }
        public int? SkipDelay { get; set; }
        public string? BackgroundColor { get; set; }
        public string? LogoUrl { get; set; }
    }
}
